<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_chambre1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border-radius:8px;
background-color: rgb(85, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Chambre1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_chambre2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border-radius:8px;
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Chambre2</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>651</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="titre">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome</string>
      </property>
     </widget>
     <widget class="QLabel" name="clock">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="date">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="image_connixon">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="text_connixson">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(85, 170, 255);</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Personne1">
       <attribute name="title">
        <string>Personne1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>281</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Suivi Sant√©</string>
        </property>
        <widget class="QLabel" name="image_temp_humaine">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/LOGIN/temp_humaine.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="image_cardiaque">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>61</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/LOGIN/logo_cardiaque.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="image_button">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>71</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/LOGIN/logo_button.jpg);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="Temperature">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_temp_humaine"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>120</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pulsation">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Pulsation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_pulsation"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>190</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="button_urgence">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Button d'urgence</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_button"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Personne2">
       <attribute name="title">
        <string>Personne2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Controle de chambre</string>
      </property>
      <widget class="QLabel" name="image_temp_humaine_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/LOGIN/logo_temperature.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="image_cardiaque_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">

border-image: url(:/LOGIN/logo_humidite.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="Temperature_chambre">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcd_temp_chambre"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="humidite">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Humidit√©</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcd_humudite"/>
        </item>
       </layout>
      </widget>
      <widget class="QRadioButton" name="radioButton_on">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-bottom-color: rgb(85, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>AUTO</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_off">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>MANUEL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/LOGIN/logo_ventilateur.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="butto_ventilateur">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>300</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">   border-radius:8px;
font: 700 10pt &quot;Segoe UI&quot;;
background-color: rgb(170, 170, 127);
</string>
       </property>
       <property name="text">
        <string>Ventilateur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonplus">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>41</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI&quot;;
border-radius:8px;

background-color: rgb(145, 170, 200);</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonplus_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>41</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI&quot;;
border-radius:8px;

background-color: rgb(145, 170, 200);</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcd_ventilateur">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 12pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/LOGIN/accueil.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LOGIN/chambre.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Accueil">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border-radius:8px;
background-color: rgb(85, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Accueil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cloud">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius:8px;
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Cloud</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_cloud">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LOGIN/logo_cloud.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/LOGIN/logo_csf.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/LOGIN/logo_issig.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sortie">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   border-radius:8px;
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/LOGIN/sortie.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
